embedded-kafka/src/test/scala/net/manub/embeddedkafka/EmbeddedKafkaCustomConfigSpec.scala

```diff
@@ -0,0 +1,34 @@
+package net.manub.embeddedkafka
+
+import scala.language.postfixOps
+
+class EmbeddedKafkaCustomConfigSpec extends EmbeddedKafkaSpecSupport with EmbeddedKafka {
+  val TWO_MEGABYTES = 2097152
```

Comment:
these should be probably be named `TwoMegabytes` and `ThreeMegabytes`  as per http://docs.scala-lang.org/style/naming-conventions.html